docker/incremental_load.sh.tpl

```diff
@@ -24,7 +24,7 @@ DOCKER="${DOCKER:-docker}"
 
 # Fetch the diff ids of the layers loaded in the docker daemon already
 IMAGES=$(${DOCKER} inspect $(${DOCKER} images -aq) |& \
-  grep sha256 | grep -v Id | cut -d'"' -f 2 | cut -d':' -f 2 | \
+  grep -E '^ +"sha256:' | grep -v Id | cut -d'"' -f 2 | cut -d':' -f 2 | \
```

Comment:
So that doesn't exactly work as is, but solves our filtering problem if we use:
```
for entry in $(docker inspect $(docker images -aq) -f '{{json .RootFS.Layers}}'); do echo -n $entry | python -mjson.tool; done | grep sha256 | cut -d'"' -f 2 | cut -d':' -f 2 | sort | uniq
```